autoptimize.php

```diff
@@ -61,8 +64,15 @@
 	$autoptimize_db_version=$autoptimize_version;
 }
 
+// Load translations from the languages directory.
```

Comment:
yep, those two lines are not needed to load the glotpress-powered 
translations.

On 09-05-16 21:14, Pablo Custo wrote:
> In autoptimize.php
> <https://github.com/futtta/autoptimize/pull/41#discussion_r62555887>:
>
>> @@ -61,8 +64,15 @@
>>  	$autoptimize_db_version=$autoptimize_version;
>>  }
>>
>> +// Load translations from the languages directory.
>
> We are talking about the same lines?
>
> |$locale = apply_filters( 'plugin_locale', $locale, 'autoptimize' );
> load_textdomain( 'autoptimize', WP_LANG_DIR . '/plugins/autoptimize-' .
> $locale . '.mo' ); |
>
> That if we remove these lines, the plugin, continue loading the language
> files from the directory |wp-content/languages/plugins/| ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/futtta/autoptimize/pull/41/files/ffdade181724d36d853639a8298b6ac02d1a0445#r62555887>
>